index.js

```diff
@@ -0,0 +1,27 @@
+function showresult(val){
+    var n1=parseFloat(document.getElementById('num1').value)
+    var n2=parseFloat(document.getElementById('num2').value)
+    
+    var c=val
```

Comment:
Naming could be better, `n1` is ok `num1` is better as the `num` at least conveys a data type. You want your variable and function names to do the heavy lifting for the person reading. `getAllOddNumbersFromAnArray` is a very helpful name. I then don't have to read the code of that function at all. Laziness FTW!